frost-core/src/traits.rs

```diff
@@ -37,7 +37,7 @@ pub trait Field: Copy + Clone {
         + Sub<Output = Self::Scalar>;
 
     /// A unique byte array buf of fixed length N.
-    type Serialization: AsRef<[u8]> + Debug + TryFrom<Vec<u8>>;
+    type Serialization: Copy + AsRef<[u8]> + AsMut<[u8]> + for<'a> TryFrom<&'a [u8]> + Debug;
```

Comment:
```suggestion
    type Serialization: AsRef<[u8]> + AsMut<[u8]> + for<'a> TryFrom<&'a [u8]> + Debug;
```

Reply:
I also need `Copy` for my DKG implementation with encryption. Maybe we could reduce it to `Clone` instead of `Copy` (to also be compatible with types like `Vec<u8>`)?